daemon/src/daemon.cpp

```diff
@@ -148,6 +151,19 @@ int main( int argc, const char* argv[] )
     cf_daemon.logging_dir = CF_LOGGING_DIR;
     cf_daemon.unix_socket_dir = CF_UNIX_DOMAIN_SOCKET_DIR;
 
+    if (getenv("CF_CRED_FILE") != NULL)
```

Comment:
Optionally, you can change CF_CRED_FILE to CF_CRED_SPEC_FILE or CF_GMSA_CRED_SPEC_FILE

Reply:
Addressed in latest updated PR, changed to CF_CRED_SPEC_FILE